tests/models/segformer/test_modeling_tf_segformer.py

```diff
@@ -336,6 +337,9 @@ def recursive_check(tuple_object, dict_object):
     def test_dataset_conversion(self):
         super().test_dataset_conversion()
 
+    def check_keras_fit_results(self, val_loss1, val_loss2, atol=2e-1, rtol=2e-1):
```

Comment:
Use higher tolerance for `TFSegformerForSemanticSegmentation`: this model 

-has `BatchNormalization` layer,
- also have several dropout layers,
- as well as a layer `TFSegformerDropPath` which has random operation during training.

These factors together cause the statistic of `moving_average` and `moving_variance` different, and we have larger validation loss.

cc @sayakpaul @amyeroberts

Reply:
I have found that we need larger tolerances for dense prediction tasks like semantic segmentation. It was the case for `TFData2VecVisionForSemanticSegmentation` as well.